projects/angular-calendar/src/date-adapters/luxon/index.ts

```diff
@@ -0,0 +1,55 @@
+import { adapterFactory as baseAdapterFactory } from 'calendar-utils/date-adapters/luxon';
```

Comment:
To make this work you'll need to add a new date adapter here:
https://github.com/mattlewis92/calendar-utils/tree/main/src/date-adapters

+ add it to the tests here:
https://github.com/mattlewis92/calendar-utils/blob/2f293a291f8fbc4da7b4022a4f562d043772755f/test/calendar-utils-utc.spec.ts#L53-L67